sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -1452,7 +1452,6 @@ class DataFrameSuite extends QueryTest with SharedSQLContext {
     withSQLConf("spark.sql.autoBroadcastJoinThreshold" -> "2") {
       val df = spark.range(100).toDF()
       val join = df.join(df, "id")
-      val plan = join.queryExecution.executedPlan
```

Comment:
I'm not sure this is without side effects, even though clearly the variable isn't used. I don't think you can remove the line, necessarily.